<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>seev::LocateJobWidget</class>
 <widget class="QWidget" name="seev::LocateJobWidget">
  <property name="windowTitle">
   <string>New Search</string>
  </property>
  <layout class="QFormLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="showSaveOptsChk">
     <property name="text">
      <string>Show Save Options</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="editAgainBut">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>TODO: Edit &amp;&amp; Search Again</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="saveBox">
     <layout class="QFormLayout" name="_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="saveBut">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="browseIconBut">
        <property name="text">
         <string>Browse Icon</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="z">
        <property name="text">
         <string>Custom Name:</string>
        </property>
        <property name="buddy">
         <cstring>saveNameEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="saveNameEdit">
        <property name="text">
         <string>New Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTableView" name="locateResLst">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="infoDispBut">
     <property name="text">
      <string>Details will be Shown Here
After Clicking Result Table.
Click here again to open it.</string>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="fetchMoreSpin">
     <property name="suffix">
      <string> Files</string>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="value">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="fetchMoreBut">
     <property name="text">
      <string>Search More</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>showSaveOptsChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>saveBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveNameEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>seev::LocateJobWidget</receiver>
   <slot>setWindowTitle(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setWindowIcon(QIcon)</slot>
 </slots>
</ui>
